test/check-application

```diff
@@ -114,11 +114,12 @@ class TestNavigator(testlib.MachineCase):
 
         b.wait_text(".breadcrumb-button:nth-of-type(1)", "/")
         b.wait_text(".breadcrumb-button:nth-of-type(2)", "home")
-        b.wait_text(".breadcrumb-button:nth-of-type(3)", "admin")
+        b.wait_text(".last-breadcrumb-button", "admin")
```

Comment:
I generally prefer to introduce classes only when they have CSS associated styles. In this case you can use the existing class as follows:

+        b.wait_text(".breadcrumb-button:last-of-type", "admin")

Reply:
If I set the `p` class back to `breadcrumb-button`, searching for `.breadcrumb-button:last-of-type` in the dev console returns the `home` button

Reply:
oh right - sorry for misleading .